dashboard/config/routes.rb

```diff
@@ -603,7 +602,7 @@
 
   get '/dashboardapi/v1/regional_partners/find', to: 'api/v1/regional_partners#find'
   get '/dashboardapi/v1/regional_partners/show/:partner_id', to: 'api/v1/regional_partners#show'
-  get '/dashboardapi/v1/pd/applications/applications_closed', to: 'api/v1/pd/applications#applications_closed'
+  get '/dashboardapi/v1/pd/application/applications_closed', to: 'pd/professional_learning_landing#applications_closed'
```

Comment:
i am not extremely familiar with rails routing stuff, let me know if there is a more appropriate way to set this up

Reply:
yes, there is a more railsy way to do it. strangely, it looks like the route you are modifying appears to be defined both in the place that you removed it from in this PR, and also here: https://github.com/code-dot-org/code-dot-org/blob/5ea77c4b19a07de2f270247d0e4a4cb8151e58b4/dashboard/config/routes.rb#L585

if your goal is to just change the route, then the railsy solution would be to update the routes.rb file so that it contains a rule like this: 
``` ruby
  concern :api_v1_pd_routes do
    namespace :pd do
      namespace :application do
        collection do
           get :applications_closed 
```
this would put the `def applications_closed` controller action back into the place you removed it from in this PR. does that solve your problem, or would the teacher still not have access?

Reply:
to be totally honest, i mostly just moved stuff around until it did work, and i'm not entirely sure why. Note that this api endpoint was added by me when we closed applications and isn't something that needs to match functionality from earlier.

Reply:
i had it defined twice before because i was following another endpoint pattern i saw in there, but i don't think that turned out to be necessary